app/javascript/controllers/toggle_field_controller.js

```diff
@@ -3,6 +3,12 @@ import { Controller } from "@hotwired/stimulus"
 export default class extends Controller {
   static targets = ["check", "field"]
 
+  connect() {
```

Comment:
this was added for the `private` checkbox that toggles the display for Auth Tokens. the default is `d-none` and this checks to see if the checkbox is already checked, and toggles it on if it is.

not sure how i feel about putting this on the `connect()` method though, since i know other uses of this controller don't operate this way (although i don't think it disturbs any of them at the moment). but i wasn't sure if there was a better way to check if a checkbox is checked on render.

Reply:
So I know it's not the greatest UX - but I'm going to advocate with consistency with Augury on this one...

Instead of "click the checkbox to add", can you just always have the extra/new token rendered on the page?  (Similar to Series->Placements and Creatives->Pingbacks in Augury).

I do want to come clean this up at some point... but parity with Augury seems like a good point to get feedback.  And really, the "delete button actually deletes" is much better than Augury already.

Reply:
maybe worded unclearly, but this check is to display/hide the entire Auth Tokens section based on `private` being checked or not, as in Publish. but I could also just display the section by default and not hide?

Reply:
Oh, maybe I misunderstood - I was thinking this was part of the "hide new token fields".

Can you just add the check to the server-side?  `class: "d-none" unless form.object.private?` or something?